src/components/Add/Add.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+
+import { Avatar, Fab, Tooltip, Typography, styled } from "@mui/material";
+import AddIcon from "@material-ui/icons/Add";
+import Modal from "@material-ui/core/Modal";
+import { Box, Stack } from "@mui/system";
+import TextField from "@material-ui/core/TextField";
+import EmojiEmotionsIcon from "@material-ui/icons/EmojiEmotions";
+import ImageIcon from "@material-ui/icons/Image";
+import PersonAddIcon from "@material-ui/icons/PersonAdd";
+import VideocamIcon from "@material-ui/icons/Videocam";
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import Button from '@material-ui/core/Button';
+import DateRangeIcon from '@material-ui/icons/DateRange';
```

Comment:
Some imports are from MUI v4 and others are from v5. Move all imports to v5 such as button etc